ui/src/Pages/Preferences/Index.jsx

```diff
@@ -28,49 +42,55 @@ function Preferences() {
     setActive(i);
   }, []);
 
+  function computeCirclePos() {
+    let containerHeight = leftProfilePic.current.clientHeight;
```

Comment:
Can you use `const` unless you're intending to change the variable value later on.